### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by lthieu96** [community/community#118764] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I can't login to vscode github copilot extension (free trial)
```
[info] [auth] Invalid copilot token: missing token: 403 
[error] [auth] Extension activation failed: "Contact Support. You are currently logged in as kakabanlaai."
```

**Comment by zyxdSTU** [comment]
I have same problem

**Comment by liteli1987gmail** [comment]
I have same problem

**Comment by davecheney** [comment, marked as answer]
@kakabanlaai @zyxdSTU @liteli1987gmail @ArtemGitCopilot i'm very sorry to report that we cannot offer further assistance via this forum. If the instructions say to contact support then you must contact support.
